steps/scan-with-appsweep/1.0.0/step.yml

```diff
@@ -0,0 +1,60 @@
+title: AppSweep Mobile Security Testing
+summary: Scan your Android app using Guardsquare's mobile application security testing
+  tool AppSweep
+description: "The build of your app is automatically uploaded to [AppSweep](https://appsweep.guardsquare.com)
+  for scanning. \n\nAppSweep is Guardsquare's free tool enabling developers to put
+  mobile security at the forefront of app development. AppSweep helps you identify
+  and fix security issues in your code and dependencies with actionable recommendations
+  and insights that will help you build more secure mobile apps.\n"
+website: https://www.guardsquare.com/appsweep-mobile-application-security-testing
+source_code_url: https://github.com/Guardsquare/bitrise-step-upload-to-appsweep
+support_url: https://github.com/Guardsquare/bitrise-step-upload-to-appsweep/issues
+published_at: 2022-01-18T12:13:50.297679995+01:00
+source:
+  git: https://github.com/Guardsquare/bitrise-step-scan-with-appsweep.git
+  commit: 85d7c53e690e51a6ed2071852f57382619057d50
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: true
+inputs:
+- appsweep_api_key: $APPSWEEP_API_KEY
+  opts:
+    category: Config
```

Comment:
If every input is in the category `Config`, then we recommend removing the category altogether, it's not mandatory. The workflow editor will display inputs in the order defined here.